docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -62,62 +56,234 @@ in 2012. This is the latest National Guideline for the Management of Acute Malnu
+
+:underline:`Health post level`
+
+ The HPs provide primary health care services such as disease prevention and control, hygiene and environmental sanitation, family health services, and health education and communication. The service provider at the HP works closely with the network of the HDA/HDG to facilitate the management of SAM and MAM. The
+ service provider diagnoses acute malnutrition and provides Outpatient Therapeutic Programme (OTP) and Targeted Supplementary Feeding Programme (TSFP) services. They also determine patients who have medical complications and refer them to the Stablisation Centre (SC).
+
+:underline:`Health center level`
+
+ The HC provides 24-hour SC for the management of SAM with medical complications. The HC also manages and provides technical support to a cluster of one to five HPs within its vicinity.
+
+:underline:`Hospital level`
+
+ Woreda, Zonal, Regional and referral Hospitals provide higher level referral points where further care can be provided to patients with SAM and more complex medical complications. The referral Hospitals have the facilities and expertise to manage situations that may require administration of oxygen, blood transfusion, and other critical care.
+
+.. _2.0:
+
+Assessing and classifying acute malnutrition
+--------------------------------------------
+
+The classification of SAM and MAM cases is derived from the 2019 National Guideline for the Management of Acute Malnutrition. [EMOH]_
+
+:download:`2019 guidelines<guidelines_2019.pdf>`
+
+.. note:: 
+  
+  - Based on the guideline, we will need to make some modelling decisions on how we classifiy SAM, MAM, NONE and think about how it may bias our results. 
+
+  - The ORs and ANDs in the classifications are not very clear or consistent in the guideline. It would be useful if we can clarify with a clinician when it should be OR/AND? 
+
+  - I think for our model we will probably classify according to the WHZ scores, but it would be good to get an idea if that over-estimates or under-estimates SAM and MAM compared to classification criteria in the guideline. 
+
+
+.. _2.1:
+
+In infants 0-6 months
++++++++++++++++++++++
 
-0-6 months
-++++++++++
+:underline:`Classify SAM with or without medical complications`
 
-Infants 0-6 months: Recent weight loss or failure to gain weight OR Ineffective feeding (attachment, position, and suckling)
+* Any grade of bilateral pitting oedema (+, ++ or +++) OR
+* WHZ < -3 zscore OR
+* Recent weight loss or failure to gain weight OR
+* Ineffective feeding (attachment, positioning and suckling) directly observed for 15-20, minutes, ideally in supervised separate area OR
+* Presence of any of the following medical complications:
 
+  - Poor appetite
```

Comment:
what combination of these is required for a diagnosis? for example, just "dehydration" wouldn't count as SAM, (right?) but all of these combined sounds very scary. 

perhaps the answer to this isnt directly applicable to our model, though

Reply:
Ya- I think for our purposes...we will just use WHZ and oedema (both gbd has) and we might need to discuss the bias of our model vs. diagnosis in real life...